### ISSUE TITLE
Accessibility (WCAG level A):  Meta tags with deployment information fail HTML 5 validation

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1473]
1.  Validate http://qa.nugettest.org/ using http://validator.w3.org/.

Result:  There'll be five validation errors, all traced to invalid name attribute values in the meta elements in the page.  (The X-UA-Compatible error is tracked separately.)
Expected:  Our pages should all validate against HTML 5.  For the new feature of adding deployment info in <meta> tags, none of the current names we're using are registered, hence we fail validation. We can do a couple things about this:
a.  We can simply insert this information in an HTML comment in the head, achieving the same goal of human-readability without upsetting the standards compliance gods, or
b.  We can attempt to get this use registered at http://wiki.whatwg.org/wiki/MetaExtensions.  I'd prefer the former, since I don't expect other sites to ever use these same tags in the manner we do.

**Comment by johnataylor** [comment]
we have a new gallery design.
